### DISCUSSION TITLE
Recover a deleted repo

### DISCUSSION THREAD

**Discussion by Talal-Saud** [community/community#118528] [Repositories]
Welcome . I have a repo and I was with a friend of mine working on a project. The repo was deleted and removed. Can you help me and take a copy of the deleted repo? I hope to find a solution because the project is very important to me

**Comment by joinemm** [comment]
https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository

**Reply by Talal-Saud** [reply]
I didn't find anything.  I was participating in a project with my friend, and I was removed and everything was deleted.  Can I restore it even if it's via GitHub support?

**Comment by devashish2024** [comment]
If your repo was a fork, then it cannot be recovered.

@Talal-Saud 
> You can only contact GitHub Support to restore a repository if you are on a paid GitHub plan. For more information about the different plans, see "[GitHub’s plans](https://docs.github.com/en/get-started/learning-about-github/githubs-plans)."

Recover personal account repo: Settings, Code and Automation, Repositories -> Deleted Repositories
Then restore it.

The steps are similir for organizations.

**Reply by Talal-Saud** [reply]
![IMG-20240417-WA0045.jpg](https://github.com/community/community/assets/42378935/2e24d956-e1bf-4826-823f-4ee1ddd58deb)

Subscribed but no repo showing
